libraries/AP_Notify/AP_Notify.cpp

```diff
@@ -304,6 +305,10 @@ void AP_Notify::add_backends(void)
 #endif
 #endif // Noise makers
 
+    // Always add AnyLED backend
+    FOREACH_I2C_EXTERNAL(b){
```

Comment:
Do you want to limit this to external, or just all buses? The internal/external bus is a much weaker distinction on some boards.

Reply:
sorry I think external bus is enough, the external led controller need external power, when I debug on cuav pixhackv3, the bus num sometimes is bus_1, sometimes bus_2, this is confused me, so I added this line. later I will show the video or pictures of this PR.

Reply:
I think doing all buses is OK.